back-end/module-client.js

```diff
@@ -1,3 +1,9 @@
+try {
+    require('dotenv').config();
+} catch (error) {
```

Comment:
Lets just use it as an direct dependency instead, this is hideous

Reply:
Yep, fixed. Much better now.